charts/v2-infra/templates/v2-relay-agent-config.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    meta.helm.sh/release-name: v2-infra
+    meta.helm.sh/release-namespace: rafay-system
+  labels:
+    app.kubernetes.io/managed-by: Helm
+  name: v2-relay-agent-config
+  namespace: rafay-system
+data:
+  clusterID: {{ .Values.cluster_id }}
+  maxDials: {{ .Values.max_dials }}
```

Comment:
Should this be configurable>
With new relay agent changes for performance, there will be more dials

Reply:
In the values.yaml, the max_dials are hardcoded to 2. However, this can still be overridden as part of `--set` flag in helm command to any custom value.